benchmark/sampling/neighborloader_bench.py

```diff
@@ -0,0 +1,105 @@
+import argparse
+import os.path as osp
+from timeit import default_timer
```

Comment:
Would be cool to add/use some functionality from `torch_geometric.profile` for this, see [here](https://pytorch-geometric.readthedocs.io/en/latest/modules/profile.html). For example, `torch_geometric.profile.timeit`.

Reply:
How about adding more profiling in next PRs? and put this PR right now as a base.

Reply:
Works for me.